.gitignore

```diff
@@ -35,5 +35,4 @@ test-results/
 # storybook
 /storybook-static
 
-# vscode
-.vscode
```

Comment:
This should still be ignored, otherwise we will get many more files added every time someone does a commit

Reply:
Okay I got it, so I should only update docs and add something like "How to debug LinkFree's Next.js code on VSCode"?

Reply:
You can add a specific config file but still ignore the `.vscode` directory because we don't want all the generated files